firmware/quadruna/VC/test/test_vcBaseStateMachineTest.h

```diff
@@ -110,14 +110,14 @@ class VcBaseStateMachineTest : public BaseStateMachineTest
     // config for heartbeat monitor (can funcs and flags)
     // VC relies on FSM, RSM, BMS, CRIT
     bool heartbeatMonitorChecklist[HEARTBEAT_BOARD_COUNT] = {
-        [BMS_HEARTBEAT_BOARD] = true, [VC_HEARTBEAT_BOARD] = false,  [RSM_HEARTBEAT_BOARD] = true,
+        [BMS_HEARTBEAT_BOARD] = true, [VC_HEARTBEAT_BOARD] = false,  [RSM_HEARTBEAT_BOARD] = false,
```

Comment:
It was my understanding that the RSM was required as it had the coolant sensors to check for over-heating conditions, and if it drops off we may not shutdown due to overtemp condition

Reply:
good call, will fix this.